Modules/State/Source/init.luau

```diff
@@ -405,6 +405,38 @@ function State.Interface.fromAttribute(object: Instance, attribute: string): Sta
 	return stateObject
 end
 
+--[=[
+	@function fromValue
+	@within State
+
+	@param object Instance & { Value: any? }
+
+	@return State
+
+	Wrapper for `State.new` however wraps around a Roblox Instance, the State object will always have the latest value.
+
+	```lua
+		local object = State.fromValue(workspace.object)
+
+		...
+	```
+]=]
+function State.Interface.fromValue(object: Instance & { Value: any? }): State
```

Comment:
```suggestion
function State.Interface.fromValue(object: ValueBase & { Value: any? }): State
```

Reply:
I had it as ValueBase at first but opted for Instance instead as it adds nothing; none of the Value classes such as `StringValue` even inherit from the `ValueBase` class. It doesn't contain the one thing all of the `Value` classes share: a `Value` property - I don't even know why it exists. Moreover, everyone is familiar with `Instance` so I want to argue it's easier comprehension, even if nothing changes Intellisense-wise.

I have no problem conforming it but I thought I'd give my thoughts 🍀